server/tsconfig.json

```diff
@@ -1,111 +1,25 @@
 {
   "compilerOptions": {
+    "sourceMap": true,
```

Comment:
give me some source why you add all that staff to config i'm curious

Reply:
i used this because i saw this on two pages we looked at together. I checked the meaning of these items and decided that i di not want to exclude soem options from these two guides at this time. But if u know some thigs about usability and you have some objection, please explain